benchmarks/benchmarks/stats.py

```diff
@@ -95,6 +95,29 @@ def time_distribution(self, distribution, properties):
     time_distribution.version = "fb22ae5386501008d945783921fe44aef3f82c1dafc40cddfaccaeec38b792b0"
 
 
+class FrozenDistribution(Benchmark):
+    params = [
+            ['multivariate_normal'],
+            ['rvs'],
+            [1000]
+    ]
+    param_names = ['distribution', 'method', 'dimension']
+
+    def setup(self, distribution, method, dim):
+        np.random.seed(4321)
+        if distribution == 'multivariate_normal':
+            mean = np.random.rand(dim)
+            eigvals = np.random.rand(dim)
+            eigvals *= dim / np.sum(eigvals)
+            cov = stats.random_correlation.rvs(eigvals)
```

Comment:
Presumably, this is just a convenient way to assemble a reasonable covariance matrix. Would any way of generating a symmetric positive semidefinite matrix do?

Reply:
Yes, this snippet is just generating a random enough covariance matrix. A sample from a Wishart distribution would've done too.

Reply:
OK, maybe add a comment explaining.